welcome-tips-rules/welcome-tips-rules.md

```diff
@@ -0,0 +1,69 @@
+# Welcome, Tips, and Rules
```

Comment:
And this one?

Reply:
Sorry but I don't know, I haven't made this change. will I revert it back?

Reply:
Another reason I think your branch is not up to date with master is that this file was deleted last week.

Reply:
I think when you were merging master in, you may have kept the wrong versions of this and ‘...sprint.md’.  delete this file and fix the other so we can merge.